doc/isomorph.xml

```diff
@@ -814,27 +814,59 @@ gap> OutNeighbours(canon);
 
 <#GAPDoc Label="IsDigraphAutomorphism">
 <ManSection>
-  <Oper Name="IsDigraphAutomorphism" Label="for a digraph and permutation"
-    Arg="digraph, x"/>
+  <Oper Name="IsDigraphIsomorphism" Arg="src, ran, x"/>
+  <Oper Name="IsDigraphAutomorphism" Arg="digraph, x"/>
   <Returns><K>true</K> or <K>false</K>.</Returns>
   <Description> 
-    This operation returns true if the permutation <A>x</A> is an automorphism
-    of the digraph <A>digraph</A>. An permutation <C>g</C> on the vertices of a
-    digraph <C>gr</C> is an automorphism if it satisfies: <C>(u, v)</C> is an
-    edge of <C>gr</C> if and only if <C>(g(u), g(v))</C> is an edge of <C>gr</C>
-    for all vertices <C>u</C> and <C>v</C> in <C>gr</C>. See also <Ref
-      Attr="AutomorphismGroup" Label="for a digraph" />.
+    <C>IsDigraphIsomorphism</C> returns <K>true</K> if the permutation or
+    transformation <A>x</A> is an isomorphism from the digraph <A>src</A> to
+    the digraph <A>ran</A>.
+    <P/>
+    
+    <C>IsDigraphAutomorphism</C> returns <K>true</K> if the permutation or
+    transformation <A>x</A> is an automorphism of the digraph <A>digraph</A>.
+    <P/>
+    
+    A permutation or transformation <A>x</A> is an <E>isomorphism</E> from a
```

Comment:
You should add that a transformation must be injective/bijective/surjective. With the definition you've got here, `Transformation([1, 1, 1])` would be an isomorphism from an empty digraph with 3 vertices to an empty digraph with 1 vertex.